client/room.html

```diff
@@ -8,7 +8,7 @@
   <link rel="icon" type="image/svg+xml" href="/i/branding/favicon.svg"/>
   <link rel="alternate icon" href="/i/branding/favicon.png"/>
   <link rel="preconnect" href="https://fonts.gstatic.com">
```

Comment:
This wouldn't be needed anymore if we also include the woff file.

Reply:
Removed.